.github/workflows/pythonpackage-linux.yml

```diff
@@ -20,6 +20,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - uses: codespell-project/actions-codespell@master
```

Comment:
You have to `pip install` and run the checker manually. Try enabling github workflow in your fork of minio-py and test it before refreshing this PR.

Reply:
Seems to work, but it was enabled for all Python versions.

![image](https://user-images.githubusercontent.com/134518/205479894-9169bc63-9f1a-414d-a055-22fa8a10fc86.png)

 Maybe we can add this as a separate workflow instead?